recommender/api/schemas/database_dump/user.py

```diff
@@ -13,6 +13,12 @@
             description="The unique identifier of a the user",
             example=1234,
         ),
+        "aai_token": fields.String(
```

Comment:
I believe it should be named "uid" not aai_token - but that's my bad. The title of the ticket was misleading. Sorry for that. Please change that everywhere